src/utils/texts.js

```diff
@@ -0,0 +1,11 @@
+export function emailToName(email) {
```

Comment:
maybe you can just list the email? some of them might be not orthodox. Keep that as a side note